packages/storage-plus/src/prefix.rs

```diff
@@ -128,7 +129,7 @@ where
 
     pub fn with_deserialization_function(
         top_name: &[u8],
-        sub_names: &[&[u8]],
+        sub_names: &[Key],
```

Comment:
Nice that (for now at least) this key hasn't leaked to `Map` and friends.

Reply:
I agree very much. I would try hard to hide that from contract developers, even if we use it everywhere in internal APIs. (And they can see it if they try to implement their own PrimaryKey type, but then we self-select for experienced devs)